services/workflows-service/prisma/schema.prisma

```diff
@@ -16,8 +16,9 @@ model User {
   password  String
   roles     Json
 
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  createdAt           DateTime              @default(now())
+  updatedAt           DateTime              @updatedAt
+  WorkflowRuntimeData WorkflowRuntimeData[]
```

Comment:
Change to `workflowRuntimeData` please. :pray:

Reply:
Thanks!